build/psr12/convert_pull_requests.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+/**
+ * This script converts Joomla Github Pull Requests to psr-12 coding standard
+ *
+ * @package        Joomla.Build
+ * @copyright  (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license        GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Set defaults
+$scriptRoot        = __DIR__;
+$repoRoot          = false;
+$prNumber          = false;
+$createPullRequest = true;
+$php               = 'php';
+$git               = 'git';
+$gh                = 'gh';
+$checkPath         = false;
+$ghRepo            = 'joomla/joomla-cms';
+$baseBranches      = '4.2-dev'; // '4.1-dev,4.2-dev,4.3-dev'; We only check for 4.2-dev
+
+$script = array_shift($argv);
+
+if (empty($argv)) {
+    echo <<<TEXT
+        Joomla! PSR-12 Converter
+        =======================
+        Usage:
+            IMPORTANT THIS SCRIPT HAVE TO BE OUTSIDE OF THE REPOSITORY
+            Best way to run this script is in a phsical copy of the original
+            The psr12_converter.php is expected in the same directory.
+
+            php {$script} [--pr=<number>] --repo=<repository rootpath>
+
+        Description:
+            The converter converts all open pull requests on github
+            to the PSR12 standard and push the changes back to github.
+
+            Flow:
+            * Load all pull requests from Github for the repository
+              $ghRepo
+            * Checkout each open pull request with the base branch matching
+              $baseBranches
+            * Merge into the checked out branch up to the psr12anchor tag
+              which includes the conversation script.
+            * Run the psr12_converter.php with the task "BRANCH"
+            * Merge up to the psr12final tag with strategy ours
```

Comment:
strategy ours?

I don't understand this bit

Reply:
Git has multiple merge strategies. And on of them is called "ours". Which actually means: drop upstream and take my version.

Reply:
The maybe quote the "ours" in the sentence with double quotes or so, so that it's clear it is a name.

Reply:
```suggestion
            * Merge up to the psr12final tag with strategy "OURS"
```

Now it matches the style of the comment above on line 47